packages/react-components/react-card/src/components/Card/Card.types.ts

```diff
@@ -1,19 +1,61 @@
+import * as React from 'react';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+/**
+ * Card refs to the root element slot.
+ */
+export type CardRefElement = HTMLDivElement | HTMLButtonElement | HTMLAnchorElement;
+
+/**
+ * Card selected event type
+ *
+ * This event is fired when a selectable card changes its selection state.
+ */
+export type CardOnSelectEvent = React.MouseEvent | React.KeyboardEvent | React.ChangeEvent;
+
+/**
+ * Data sent from the selection events on a selectable card.
+ */
+export type CardOnSelectData = {
+  selected: boolean;
+};
+
 /**
  * Slots available in the Card component.
  */
 export type CardSlots = {
   /**
    * Root element of the component.
    */
-  root: Slot<'div'>;
+  root: Slot<'div', 'a' | 'button'>;
+
+  /**
+   * Select element represents a checkbox.
+   */
+  select?: Slot<'div', 'input'>;
```

Comment:
if it represents a checkbox why not call it a `checkbox` slot ? 😅

Reply:
Because it is to represent a "selectable" card, so that is why we chose `select`, `selected` and `onCardSelect`. In most cases this can be a checkbox but can also receive a radio.